ansys/mapdl/core/_version.py

```diff
@@ -8,7 +8,7 @@
 """
 
 # major, minor, patch
-version_info = 0, 60, "dev0"
+version_info = 0, 60, "devpsi0"
```

Comment:
We generally don't increment the dev tag in PRs unless we're bumping the latest release branch. That being said, our version should actually be:

```suggestion
version_info = 0, 61, "dev0"
```

as the stable release on PyPI is [0.60.5](https://pypi.org/project/ansys-mapdl-core/) with the latest release branch [release/0.60](https://github.com/pyansys/pymapdl/tree/release/0.60).